src/ft2font_wrapper.cpp

```diff
@@ -291,7 +291,7 @@ static unsigned long read_from_file_callback(FT_Stream stream,
             return 1;  // Non-zero signals error, when count == 0.
         }
     }
-    return n_read;
+    return PyLong_AsUnsignedLong(PyLong_FromSsize_t(n_read));
```

Comment:
This change is new from after I reviewed; it leaks the `PyLong` you just created. Anyway, I don't know why you need a `PyLong` and can't just cast it in C++?